db/migrate/20161010202701_add_commenter_fields_to_flags_and_votes.rb

```diff
@@ -0,0 +1,19 @@
+class AddCommenterFieldsToFlagsAndVotes < ActiveRecord::Migration
+  def change
+    add_column :flyover_comments_flags, :commenter_id, :integer
+    add_column :flyover_comments_flags, :commenter_type, :string
+    add_column :flyover_comments_votes, :commenter_id, :integer
+    add_column :flyover_comments_votes, :commenter_type, :string
```

Comment:
I think it probably makes more sense to make these flagger and voter rather than commenter